src/ome_zarr_models/v04/multiscales.py

```diff
@@ -16,13 +17,12 @@
     _build_transforms,
     _ndim,
 )
+from ome_zarr_models.v04.omero import Omero  # noqa: TCH001
```

Comment:
This noqa looks suspicious to me 🙈  - can you either explain it here, remove it, or just open an issue to track checking these later (happy to stick noqa in for development velocity now, as long as we remember to check later!)

Reply:
ruff doesn't know that pydantic needs the type annotations at runtime, so without importing `Omero` at runtime, models with fields annotated with `Omero` are broken

Reply:
(and without the noqa, ruff will stick `Omero` in the `if TYPE_CHECKING` block)